src/frontend/components/SøknadsSteg/OmDeg/Personopplysninger.tsx

```diff
@@ -6,51 +6,48 @@ import { BodyShort, Label } from '@navikt/ds-react';
 
 import { useApp } from '../../../context/AppContext';
 import { useSpråk } from '../../../context/SpråkContext';
+import { ESanitySteg } from '../../../typer/sanity/sanity';
 import { genererAdresseVisning } from '../../../utils/adresse';
-import { hentSivilstatusSpråkId, landkodeTilSpråk } from '../../../utils/språk';
-import SpråkTekst from '../../Felleskomponenter/SpråkTekst/SpråkTekst';
-
-import { omDegPersonopplysningerSpråkId } from './spørsmål';
+import { landkodeTilSpråk, sivilstandTilSanitySivilstandApiKey } from '../../../utils/språk';
 
 export const Personopplysninger: React.FC = () => {
     const { valgtLocale } = useSpråk();
-    const { søknad } = useApp();
+    const { søknad, tekster, plainTekst } = useApp();
+
+    const {
+        [ESanitySteg.OM_DEG]: { ident, statsborgerskap, sivilstatus, adresse },
+        [ESanitySteg.FELLES]: { frittståendeOrd },
+    } = tekster();
 
     const søker = søknad.søker;
 
+    const statsborgerskapTekst = søker.statsborgerskap
+        .map((statsborgerskap: { landkode: Alpha3Code }) =>
+            landkodeTilSpråk(statsborgerskap.landkode, valgtLocale)
+        )
+        .join(', ');
+
+    const sivilStatusTekst = plainTekst(
+        frittståendeOrd[sivilstandTilSanitySivilstandApiKey(søker.sivilstand.type)]
+    );
+
     return (
         <>
             <div>
-                <Label as="p">
-                    <SpråkTekst id={'felles.fødsels-eller-dnummer.label'} />
-                </Label>
+                <Label>{plainTekst(ident)}</Label>
```

Comment:
Her burde vi bruke noe annet enn kun `Label`, som kompilerer som en tom `<label>`. Kanskje gå for ´<BodyShort weight="semibold||bold">` her?